<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PictureViewerThumbnailsClass</class>
 <widget class="QMainWindow" name="PictureViewerThumbnailsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>1024</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PictureViewerThumbnails</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,10">
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="styleSheet">
         <string notr="true">QGroupBox {
    border: 1px solid #DDD;
    border-radius: 4px;
    font: bold 14px;
    color: #666;
    background-color: rgb(202, 202, 202);
    padding: 10px; 
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    left: 10px;
    padding: 0 5px;
}</string>
        </property>
        <property name="title">
         <string>文件目录</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QTreeView" name="treeView_categoryTree">
           <property name="styleSheet">
            <string notr="true">/* 基础树形视图 */
QTreeView {
    border: 1px solid #AAAAAA;
    border-radius: 3px;
    background: #FFFFFF;
    color: #444444;
    font-size: 14px;
    outline: 0;
    padding: 2px;
    show-decoration-selected: 1;
    indentation: 24px;  /* 层级缩进 */
}

/* 树形分支线 */
QTreeView::branch {
    background: #FFFFFF;
    border-left: 1px dotted #DDDDDD;  /* 层级连接线 */
    margin-left: 12px;
}

QTreeView::branch:has-siblings {
    border-left: none;
}

/* 展开/折叠图标 */
QTreeView::branch:closed:has-children {
    border-image: url(:/icons/tree_closed_gray.svg) 0;
}

QTreeView::branch:open:has-children {
    border-image: url(:/icons/tree_open_gray.svg) 0;
}

/* 列表项通用样式 */
QTreeView::item {
    height: 28px;
    padding: 4px 8px;
    border-radius: 2px;
    margin: 1px 0;
    border: 1px solid transparent;
}

/* 悬停效果 */
QTreeView::item:hover {
    background: #F8F8F8;
    border-color: #888888;
}

/* 选中状态 */
QTreeView::item:selected {
    background: #666666;
    color: #FFFFFF;
    border-color: #555555;
}

/* 焦点状态 */
QTreeView::item:selected:focus {
    background: qlineargradient(
        x1:0, y1:0, x2:0, y2:1,
        stop:0 #666666, 
        stop:1 #555555
    );
}

/* 禁用状态 */
QTreeView:disabled {
    border-color: #DDDDDD;
    color: #AAAAAA;
}

QTreeView::item:disabled {
    background: #F5F5F5;
    color: #AAAAAA;
}

/* 紧凑型变体 */
QTreeView.compact {
    font-size: 13px;
    indentation: 20px;
}

QTreeView.compact::item {
    height: 24px;
    padding: 2px 6px;
}

/* 交替行颜色 */
QTreeView::item:alternate {
    background: #FAFAFA;
}

/* 表头样式 */
QHeaderView {
    background: #FFFFFF;
}

QHeaderView::section {
    background: #F8F8F8;
    border: 1px solid #DDDDDD;
    padding: 6px 8px;
    color: #666666;
    font-weight: 500;
}

/* 排序指示器 */
QHeaderView::down-arrow {
    image: url(:/icons/sort_desc_gray.svg);
}

QHeaderView::up-arrow {
    image: url(:/icons/sort_asc_gray.svg);
}</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="styleSheet">
         <string notr="true">QGroupBox {
    border: 1px solid #DDD;
    border-radius: 4px;
    font: bold 14px;
    color: #666;
    background-color: rgb(202, 202, 202);
    padding: 10px; 
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    left: 10px;
    padding: 0 5px;
}</string>
        </property>
        <property name="title">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="styleSheet">
       <string notr="true">QGroupBox {
    border: 1px solid #DDD;
    border-radius: 4px;
    font: bold 14px;
    color: #666;
    background-color: rgb(161, 161, 161);
    padding: 2px; 
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    left: 10px;
    padding: 0 5px;
}</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="pbtn_exit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    padding: 6px 14px;
    border: 2px solid #CCC;
    border-radius: 4px;
    background-color: white;
    color: #444;
    font-size: 30px; 
}

QPushButton:hover {
    border-color: #999;
    background-color: #F5F5F5;
}

QPushButton:pressed {
    border-color: #777;
    background-color: #EEE;
}</string>
         </property>
         <property name="text">
          <string>退出</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbtn_deleteTotal">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    padding: 6px 14px;
    border: 2px solid #CCC;
    border-radius: 4px;
    background-color: white;
    color: #444;
    font-size: 30px; 
}

QPushButton:hover {
    border-color: #999;
    background-color: #F5F5F5;
}

QPushButton:pressed {
    border-color: #777;
    background-color: #EEE;
}</string>
         </property>
         <property name="text">
          <string>全部删除</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbtn_delete">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    padding: 6px 14px;
    border: 2px solid #CCC;
    border-radius: 4px;
    background-color: white;
    color: #444;
    font-size: 30px; 
}

QPushButton:hover {
    border-color: #999;
    background-color: #F5F5F5;
}

QPushButton:pressed {
    border-color: #777;
    background-color: #EEE;
}</string>
         </property>
         <property name="text">
          <string>删除</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbtn_bigger">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    padding: 6px 14px;
    border: 2px solid #CCC;
    border-radius: 4px;
    background-color: white;
    color: #444;
    font-size: 30px; 
}

QPushButton:hover {
    border-color: #999;
    background-color: #F5F5F5;
}

QPushButton:pressed {
    border-color: #777;
    background-color: #EEE;
}</string>
         </property>
         <property name="text">
          <string>放大</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbtn_smaller">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    padding: 6px 14px;
    border: 2px solid #CCC;
    border-radius: 4px;
    background-color: white;
    color: #444;
    font-size: 30px; 
}

QPushButton:hover {
    border-color: #999;
    background-color: #F5F5F5;
}

QPushButton:pressed {
    border-color: #777;
    background-color: #EEE;
}</string>
         </property>
         <property name="text">
          <string>缩小</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbtn_prePicture">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    padding: 6px 14px;
    border: 2px solid #CCC;
    border-radius: 4px;
    background-color: white;
    color: #444;
    font-size: 30px; 
}

QPushButton:hover {
    border-color: #999;
    background-color: #F5F5F5;
}

QPushButton:pressed {
    border-color: #777;
    background-color: #EEE;
}</string>
         </property>
         <property name="text">
          <string>上一张</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbtn_nextPicture">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    padding: 6px 14px;
    border: 2px solid #CCC;
    border-radius: 4px;
    background-color: white;
    color: #444;
    font-size: 30px; 
}

QPushButton:hover {
    border-color: #999;
    background-color: #F5F5F5;
}

QPushButton:pressed {
    border-color: #777;
    background-color: #EEE;
}</string>
         </property>
         <property name="text">
          <string>下一张</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
